src/utils/auth/user.mock.ts

```diff
@@ -0,0 +1,14 @@
+import config from '../../config';
+import { Types } from 'mongoose';
+import { sign } from 'jsonwebtoken';
+import { ROLE } from '../../permissions/user.interface';
+
+export function ValidTokenMock() {
+  const user = {
+    id: new Types.ObjectId(),
+    personId: new Types.ObjectId(),
+    role: ROLE.READ,
```

Comment:
Should be `role: ROLE.WRITE` ? (To allow all requests)

Reply:
yep, I changed it for testing. done